README.md

```diff
@@ -204,46 +191,48 @@ spec:
+Using the Kogito CLI, specify how much memory and CPU your service requires:
 
 ```bash
 kogito deploy onboarding-service https://github.com/kiegroup/kogito-examples --context-dir onboarding-example/onboarding --native --build-limits memory=12Gi cpu=2  
 ```
 
-For more information, see [Native X JVM Builds](#native-x-jvm-builds).
+For more information, see [Native X JVM builds](#native-x-jvm-builds).
+
+## Kogito Data Index Service deployment
+
 
-## Deploy Data Index Service
+The Kogito Operator can deploy the [Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service) as a [Custom Resource](deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml) (`KogitoDataIndex`).
 
-The Kogito Operator is able to deploy the [Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service) as a [Custom Resource](deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml) (`KogitoDataIndex`). 
+The Data Index Service depends on Kafka, so you must manually deploy an Apache Kafka Cluster, such as [Strimzi](https://strimzi.io/).
 
-Since Data Index Service depends on Kafka, it's necessary to manually deploy an Apache Kafka Cluster (we use [Strimzi](https://strimzi.io/)) in the same namespace.
- 
-Data Index also depends on Infinispan, but since 0.6.0 version, Infinispan Server is *automagically* deployed for you.
+The Data Index Service also depends on Infinispan, but starting in version 0.6.0 of the Kogito Operator, Infinispan Server is automatically deployed for you.
 
-| :information_source: It's planned for future releases that the Kogito Operator will deploy a Kafka cluster when deploying the Data Index Service. |
+
+| :information_source: In a future release, the Kogito Operator will deploy a Kafka cluster when deploying the Data Index Service. |
 | --- |
 
-### Deploy Infinispan
+### Deploying Infinispan
 
-If you plan to have the Data Index to connect to the Infinispan server deployed within the same namespace, Kogito Operator can handle this deployment for you.
+If you plan to use the Data Index Service to connect to an Infinispan Server instance deployed within the same namespace, the Kogito Operator can handle this deployment for you.
 
-When installing the Kogito Operator from OperatorHub, the Infinispan Operator will be available in the same namespace. Otherwise, if you don't have OperatorHub or OLM available in your cluster, you'll have to [subscribe to Infinispan Operator](https://infinispan.org/infinispan-operator/master/documentation/asciidoc/titles/operator.html#deploying_operator_manually) manually.
+When you install the Kogito Operator from OperatorHub, the Infinispan Operator is available in the same namespace. If you do not have access to OperatorHub or OLM in your cluster, you can [manually deploy the Infinispan Operator](https://infinispan.org/infinispan-operator/master/documentation/asciidoc/titles/operator.html#deploying_operator_manually).
```

Comment:
When you install the Kogito Operator from OperatorHub, the Infinispan Operator should be installed in the same namespace when first installing the Data Index Service.
@ricardozanini for confirmation

Reply:
Actually when installing the Operator from Hub, the OLM will install the Infinispan operator for us since it’s a requirement in the CSV file. So I’d change to “will install” instead of “should”. @sterobin wdyt?

Reply:
Will use "is" though, because best practice to avoid future tense as much as possible. But @radtriste @ricardozanini, is Infinispan installed when the Operator is installed or when the Data Index Service is installed? @radtriste wording basically says both, which confuses me.

I've written just as this for now:

>When you install the Kogito Operator from OperatorHub, the Infinispan Operator is installed in the same namespace.

Reply:
@sterobin when Kogito Operator is installed via OperatorHub, Infinispan Operator is also installed. When Data Index is deployed with `UseKogitoInfra` parameter set to `true`, Infinispan Server is deployed to serve as the persistence actor in the infrastructure.

Reply:
@sterobin do you guys have a document for writing best practices? Even though I know I lack English knowledge, will be nice to have something like this to have in mind when writing READMEs.

Reply:
I've added what you've suggested and rearranged this bit as follows. If this is still not right or not what you wanted, please correct it directly so that it's accurate:

> When you install the Kogito Operator from OperatorHub, the Infinispan Operator is installed in the same namespace. When you install the Data Index Service with the `UseKogitoInfra` parameter set to `true`, Infinispan Server is deployed to serve as the persistence actor in the infrastructure. 

> If you do not have access to OperatorHub or OLM in your cluster, you can [manually deploy the Infinispan Operator](https://infinispan.org/infinispan-operator/master/documentation/asciidoc/titles/operator.html#deploying_operator_manually).
```

@ricardozanini , regarding style, Red Hat technical documentation standards are based on the [IBM Style Guide](https://www.amazon.com/IBM-Style-Guide-Conventions-Writers/dp/0132101300/ref=sr_1_1?keywords=IBM+style+guide&qid=1574270099&s=books&sr=1-1) (coincidentally), of which there's an abbreviated [version online for developers](https://www.ibm.com/developerworks/library/styleguidelines/index.html). There are various tech comm style guides that I've had to ingest (Microsoft, Sun Microsystems, IBM, etc.) but for your needs, that online IBM version should be sufficient. 

I also strongly recommend to anyone I talk to about this to look into the [Global English Style Guide](https://www.amazon.com/Global-English-Style-Guide-Documentation/dp/1599946572/ref=sr_1_1?keywords=global+english+style+guide&qid=1574270270&s=books&sr=1-1). An invaluable asset for mass communicators, particularly in open source communities. I became good friends with the author, now retired from SAS.

Reply:
Gosh! How many valuable information there! Many thanks for this, @sterobin